main.tf

```diff
@@ -4,8 +4,6 @@ resource "castai_eks_cluster" "my_castai_cluster" {
   name       = var.aws_cluster_name
 
   delete_nodes_on_disconnect = var.delete_nodes_on_disconnect
-  access_key_id              = var.aws_access_key_id
```

Comment:
We should remove them from `variables.tf` I think

Reply:
removed